usmqe_tests/conftest.py

```diff
@@ -388,3 +389,25 @@ def fill_memory():
         SSH[host].run(teardown_cmd)
         return request.param
     return measure_operation(fill_memory)
+
+
+@pytest.fixture
+def workload_stop_nodes():
+    """
+    Test ran with this fixture have to have use fixture `ansible_playbook`
+    and markers before this fixture is called:
+
+    @pytest.mark.ansible_playbook_setup("test_setup.stop_tendrl_nodes.yml")
+    @pytest.mark.ansible_playbook_teardown("test_teardown.stop_tendrl_nodes.yml")
+
+    Returns:
+        dict: contains information about `start` and `stop` time of wait
+        procedure and as `result` is used number of nodes.
+    """
+    # wait for tendrl to notice that nodes are down
+    time.sleep(240)
```

Comment:
Let's add a log line, which states what you stated in the comment: wait for tendrl to notice that nodes are down. It would make it more clear what's going on when one checks logs while the test is running.